lib/host.py

```diff
@@ -52,6 +53,18 @@ def host_data(hostname_or_ip):
 class Host:
     xe_prefix = "host"
 
+    # Must be kept in sync. Data extraction is automatic, no conversion from str is done.
+    BLOCK_DEVICES_FIELDS = "NAME,KNAME,PKNAME,SIZE,LOG-SEC,TYPE"
```

Comment:
maybe replace the explicit list with
```python
','.join(k.upper() for k in BlockDeviceInfo.__annotations__.keys())
```
?

Reply:
Good point, I was too lazy to dig that line at the time, thx :)